config.py

```diff
@@ -17,9 +17,9 @@
 BOT_ASYNC = True
 BOT_ASYNC_POOLSIZE = 10
 BOT_IDENTITY = {'token': secrets.get('Slackbot', 'token')}
-BOT_ADMINS = ('@mattreed79', '@paulkarayan')
```

Comment:
we need a more general way of dealing with this. i suggest we put them in via the .secrets
thoughts?

Reply:
I recommend a config file.  Secrets will override what's in the config file, where given.